test/param/ferguson/tryresolve-user.chpl

```diff
@@ -0,0 +1,92 @@
+
+
+proc hi0() {
+  writeln("hi");
+}
+
+proc hi1(x:int) {
+  writeln("hi", x);
+}
+
+proc hi2(x:int, y) {
+  writeln("hi", x);
+}
+
+
+var x:bool;
+
+x = canResolve("hi0");
+assert(x);
```

Comment:
Does the use of 'x' in this test add any value / work around some issue compared to doing 'assert(canResolve(...))'?